cookbooks/cdo-nginx/templates/default/nginx.conf.erb

```diff
@@ -0,0 +1,53 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 1024;
+  use epoll;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  tcp_nodelay on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
+  proxy_redirect off;
+  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  client_max_body_size 4G;
+  keepalive_timeout 5;
```

Comment:
Not sure I remember specifically how I chose this value, but the example [`nginx.conf`](http://unicorn.bogomips.org/examples/nginx.conf) from Unicorn upstream recommends 5 seconds with the following comment:

```
    # ~2 seconds is often enough for most folks to parse HTML/CSS and
    # retrieve needed images/icons/frames, connections are cheap in
    # nginx so increasing this is generally safe...
    keepalive_timeout 5;
```

I'm fine increasing this to 30, it doesn't sound like lingering connections are as big of a problem to nginx as they are to unicorn.